Day2/app.js

```diff
@@ -0,0 +1,9 @@
+const app=require('./mockData')
+const express=require('express')
+const value=express()
```

Comment:
change the name to app or express app, value variables is not suited here and for mockData change that name also

Reply:
okay

Reply:
comment resolved.